LICENSE

```diff
@@ -0,0 +1,17 @@
+PipeVal is licensed under the GNU General Public License
```

Comment:
Swap the contents of LICENSE and COPYING. LICENSE is meant to contain the full license, while COPYING includes the license copyright clause and copying instructions.

Reply:
I think it's the other way around
https://www.gnu.org/licenses/gpl-howto.en.html
https://github.com/daviddao/CellProfiler-REST/tree/168992170aebe9925d70347d422163bd2ee5d4c0

Reply:
Good finds. Perhaps surprisingly, it seems [every variation](https://github.com/search?p=1&q=COPYING+LICENSE&type=Commits) under the sun is used. I generally reference [the Linux kernel](https://github.com/torvalds/linux), but even that is inconsistent with [Git](https://github.com/git/git) - Torvald's other major software project. I reference [dotnet](https://github.com/dotnet/runtime) and [Python](https://github.com/python/cpython) as well. So many differences!

Note that GitHub uses a Ruby Gem called [Licensee](https://github.blog/2016-09-21-license-now-displayed-on-repository-overview/), which looks specifically for `LICENSE` [files](https://github.com/licensee/licensee). Granted, the GitHub and GPL conventions are inconsistent. It's also worth noting that, when creating a new repository, if you select to add a license file, GitHub stores the contents in the file `LICENSE`.

So with all that information, it seems the decision is really ours. Personally I think we should follow the GitHub convention to stay consistent within our ecosystem.

Reply:
@aholmes Let me know if this is good to go!